README.md

```diff
@@ -87,6 +87,10 @@ Available options/variables and their default values:
 | GOG_EMAIL     	|         	| GOG email for login. Overrides EMAIL.                                  	|
 | GOG_PASSWORD  	|         	| GOG password for login. Overrides PASSWORD.                            	|
 | GOG_NEWSLETTER	| 0       	| Do not unsubscribe from newsletter after claiming a game if 1.         	|
+| GOG_GIVEAWAY	  | 1       	| Claims giveaway game(s). Is enabled by default. |
+| GOG_FREEGAMES	  | 0      	| Claims other free games that are not demos or prologue games. Is disabled by default.         	|
```

Comment:
Would remove 'Is enabled/disabled by default.' since the default value is already there.